bundles/specmate-dbprovider-api/src/com/specmate/dbprovider/api/IDBProvider.java

```diff
@@ -26,9 +26,9 @@
 	public void registerDBConfigChangedCallback(DBConfigChangedCallback cb);
 
 	/**
-	 * @return the configured database resource string
+	 * Cancels the registration
 	 */
-	public String getResource();
+	public void unregisterDBConfigChangedCallback(DBConfigChangedCallback cb);
 
 	/**
 	 * @return the configured database repository string
```

Comment:
I think also the getRepository() method can be removed from the interface (and in the implementing classes down the line, including their configuration), as this data is now provided in the SpecmateCDOServerConfig. There it is however only half implemented (instance variable defined, read from configService and write to configurationAdmin is missing).
This would then make a clear separation of configuration of CDO vs. Databases.